Header header

int32 job_number

bool is3d

int32 number_of_items

int32[] categories
float32[] scores
